airbyte-integrations/connectors/source-mysql/src/test/java/io/airbyte/integrations/source/mysql/MySqlSourceOperationsTest.java

```diff
@@ -302,5 +302,4 @@ protected void executeQuery(final String query) {
       throw new RuntimeException(e);
     }
   }
-
```

Comment:
nit: is this auto-formatting causing mischief?

Reply:
This one was me. I'll update it.